src/plugins/github/api.test.js

```diff
@@ -0,0 +1,76 @@
+// @flow
+
+import {parse} from "./parser";
+import exampleRepoData from "./demoData/example-repo.json";
+import {Repository, Issue, PullRequest, Comment, Author} from "./api";
+import {
+  AUTHOR_NODE_TYPE,
+  COMMENT_NODE_TYPE,
+  ISSUE_NODE_TYPE,
+  PULL_REQUEST_NODE_TYPE,
+} from "./types";
+describe("GitHub porcelain API", () => {
+  const graph = parse("sourcecred/example-repo", exampleRepoData);
+  const repo = new Repository("sourcecred/example-repo", graph);
+
+  it("Issue", () => {
+    const issue = repo.issueOrPRByNumber(1);
+    if (issue == null) {
+      throw new Error("Issue reaching issue!");
+    }
+    expect(issue.title()).toBe("An example issue.");
+    expect(issue.body()).toBe("This is just an example issue.");
+    expect(issue.number()).toBe(1);
+    expect(issue.type()).toBe(ISSUE_NODE_TYPE);
+    expect(issue.url()).toBe(
+      "https://github.com/sourcecred/example-repo/issues/1"
+    );
+    expect(issue.type()).toMatchSnapshot();
+    expect(issue.authors()[0].login()).toBe("decentralion");
+  });
```

Comment:
Did you want to test the `issue.address()`, as with the other object types?

Reply:
Fixed.